spotbugs/src/main/java/edu/umd/cs/findbugs/detect/DateFormatStringChecker.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * FindBugs - Find bugs in Java programs
```

Comment:
New files should not use a FindBugs copyright notice.

Reply:
@ramraval It's great that you removed the FindBugs copyright, but the file should contain a copyright notice, the SpotBugs one.